### DISCUSSION TITLE
Welcome to figma-vr-unity-converter Discussions!

### DISCUSSION THREAD

**Discussion by marcgc21** [uramakilab/figma-vr-unity-converter#23] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by elenamolero** [comment]
Hi, I am Elena, a last year software engineer student who loves to contribute in Open source projects. In my university I am member of an Open Source Code Group called  "Aula de Software Libre de la Universidad de Córdoba". I have worked with them coding and doing designs in Figma and for merchandising. I really love Figma, I have a portfolio which I would be glad to share with you. I have prepare the enviroments but I don't know how to start contribuing woth you. Where is the discrod chanel?
